src/main/kotlin/uk/gov/dwp/dataworks/services/ConfigurationResolver.kt

```diff
@@ -63,6 +63,7 @@ enum class ConfigKey(val key: String, val isList: Boolean) {
     ECS_CLUSTER_NAME("orchestrationService.ecs_cluster_name", false),
     USER_CONTAINER_URL("orchestrationService.user_container_url", false),
     USER_CONTAINER_PORT("orchestrationService.user_container_port", false),
+    JUPYTER_S3_ARN("orchestrationService.jupyterhub_bucket_arn", false),
```

Comment:
This env var should also be added in the terraform deployment

Reply:
I just had a look and, as far as I can see, it is. Have I missed something?

Reply:
Yeah, my bad, wasn't looking on the right branch locally 🤦